### PR TITLE
Client go

### PR DESCRIPTION
**Pull request by chuckha**
Fixes: #196

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -311,41 +245,35 @@
   branch = "v2"
   name = "gopkg.in/yaml.v2"
   packages = ["."]
-  revision = "eb3733d160e74a9c7e442f435eb3bea458e1d19f"
+  revision = "d670f9405373e636a5a2765eea47fac0c9bc91a4"
 
 [[projects]]
-  branch = "release-1.8"
+  branch = "master"
```
Comment (timothysc) [inline Gopkg.lock]: You're going to want to pin these to the branch version.

Comment (timothysc) [reply Gopkg.lock]: Then call dep-ensure.

Comment (chuckha) [reply Gopkg.lock]: ahh ok, let me try agian

**Comment by chuckha** [comment]
@timothysc Updated with pinned k8s versions, please take a look.

File: Gopkg.toml [hunk 2]
```diff
@@ -62,15 +62,15 @@
   name = "github.com/viniciuschiele/tarx"
 
 [[constraint]]
-  branch = "master"
+  branch = "release-1.9"
```
Comment (sttts) [inline Gopkg.toml]: tag kubernetes-1.9.0 if you pin to v6.0.0.

**Pull request merged**